recce.ltx

```diff
@@ -2303,7 +2302,7 @@ A \dfn{context free grammar} (CFG) is a 4-tuple,
 \begin{equation}
 \begin{gathered}
 \text{$[\Vsymset{nt}, \Vsymset{term}, \Vruleset{rules}, \Vsym{accept}]$ such that} \\
-\Vsymset{nt} \cup \Vsymset{term} = \emptyset, \\
```

Comment:
This would be a trivial grammar, where terminals and non-terminals are both the empty set :)

Reply:
It would have made a lot of my theorems vacuously true, shortening the proofs considerably. :-)